src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -571,7 +572,7 @@
 
 								<CheckBox
 									x:Name="SelectionCheckbox"
-									Width="{Binding ElementName=SelectAllColumn, Mode=OneWay, Path=Width}"
```

Comment:
I think we can probably even remove this line given the visibility setter at the bottom.

Reply:
It seems like you are right.  `CheckBox.Width` is actually the width of the label of the `CheckBox`. So I Remove this line.

Reply:
Oh yes, there is no need for it. I will remove it.